### ISSUE TITLE
Roadmap in relation to the Helm Charts

### ISSUE THREAD

**Issue by Ornias1993** [k8s-at-home/container-images#25]
Firstoff:
Design looks awesome, personally hate the overcomplicated ever-breaking s6 overlay.

But: What's the plan?
Is this already natively copy-pasta replacement for the images used in the Helm charts?
Is the goal to evenually be a replacement for us with the helmcharts (aka fully replace LinuxServer containers)?
If is, whats the timeframe we should reasonably expect for it to hit the helmcharts?

Is this a valid issue?
Yes.
TLDR:
Maybe a little readme update is in order? ;-)

**Comment by onedr0p** [comment]
> Is this already natively copy-pasta replacement for the images used in the Helm charts?

Yes, pretty much. I and a few others have been using them for a couple months.

> Is the goal to evenually be a replacement for us with the helmcharts (aka fully replace LinuxServer containers)?

I'd like to see that happen, but because these aren't as user friendly, I don't know if we will ever make it a default in the charts.

> If is, whats the timeframe we should reasonably expect for it to hit the helmcharts?

You can use it now!

> Maybe a little readme update is in order?

For sure, I am just terrible at documentation :)

**Comment by Ornias1993** [comment]
Thanks for the headsup! :)
The userfriendlyness of the LinuxServer containers comes at a cost imho, like: not being able to setup read-only-root.

Hard at work porting some k8s-at-home containers over to the new TrueNAS SCALE App UI, as the UI is only going to show a limited subset of options anyway... I think this would work just fine, but i'll let it crystalise for a few months first 👍 
(no need to rush stacking beta ontop of beta ;-) )

> For sure, I am just terrible at documentation :)
No worries, arn't we all 🥇

**Comment by onedr0p** [comment]
Feel free to test them out and provide some feedback :)

**Comment by Ornias1993** [comment]
Will do.
And put my money where my mouth was ;-)
